data/assets/scene/milkyway/objects/orionnebula/cluster.asset

```diff
@@ -23,18 +23,9 @@ local OrionClusterStars = {
     },
     GUI = {
         Name = "Orion Nebula Star Cluster",
-        Path = "/Milky Way/Stars"
-    }
-}
-
-assetHelper.registerSceneGraphNodesAndExport(asset, { OrionClusterStars })
-
-
-asset.meta = {
-    Name = "Orion Nebula Star Cluster",
-    Version = "1.0",
-    Description = [[ In order to have an accurate depiction of the Orion nebula, we need 
-to include the star cluster that was birthed from it. We turned to a study of the 
+        Path = "/Milky Way/Stars",
```

Comment:
Does it makes sense to put all of the orion things into `/Milky Way/Orion` instead?

Reply:
way better